libraries/chain/block_trace.cpp

```diff
@@ -42,7 +66,7 @@ namespace eosio { namespace chain {
       for(const auto& rt: region_traces ) {
```

Comment:
Since we are here and doing this, do we want to add in an indirection for region parallelism?